src/vtbackend/Terminal.cpp

```diff
@@ -1262,9 +1262,9 @@ void Terminal::verifyState()
     Require(*_currentMousePosition.column < *thePageSize.columns);
     Require(*_currentMousePosition.line < *thePageSize.lines);
 
-    Require(_hostWritableStatusLineScreen.pageSize() == _indicatorStatusScreen.pageSize());
-    Require(_hostWritableStatusLineScreen.pageSize().lines == LineCount(1));
-    Require(_hostWritableStatusLineScreen.pageSize().columns == _settings.pageSize.columns);
+    // Require(_hostWritableStatusLineScreen.pageSize() == _indicatorStatusScreen.pageSize());
+    // Require(_hostWritableStatusLineScreen.pageSize().lines == LineCount(1));
+    // Require(_hostWritableStatusLineScreen.pageSize().columns == _settings.pageSize.columns);
```

Comment:
Why did you comment them out? Did they fail? Still? Maybe they fail for a reason, that we should address?